### PR TITLE
Implemented the 2nd puzzle and fix issues

### PR DESCRIPTION
**Pull request by EnzoGuillouche**
The 2nd puzzle is fully implemented into the game.
Some issues have been fixed.

### CODE REVIEW

**Review by MistzSoftware — DISMISSED** [review]
It seems ok for me, just the comment below to clarify but it's really minor

File: Game/3d-game-project/Dialogue/GivenCard.dtl [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[style name="Style"]
+Scientist (Happy): Ookay, this gave you access to the panel.
```
[the remark this answers is not in the document]
Comment (MistzSoftware) [reply Game/3d-game-project/Dialogue/GivenCard.dtl:2]: explain orally, everything is fine.

**Review by MistzSoftware — APPROVED** [review]
Perfect bebou! ❤️

ready to merge

**Review by Evan-UHRING — COMMENTED** [review]
Screenshot 2024-12-18 at 12 58 24

it crashes after finishing the 2nd puzzle when i try to take the teleporter to go back to the hub

**Review by Evan-UHRING — APPROVED** [review]
👍​

**Pull request merged**